cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/CompileCommand.java

```diff
@@ -155,8 +155,8 @@ public void execute() {
             if (!ProjectDirs.isProject(this.sourceRootPath)) {
                 Path findRoot = ProjectDirs.findProjectRoot(this.sourceRootPath);
                 if (null == findRoot) {
-                    throw LauncherUtils.createLauncherException("you are trying to compile a ballerina project but " +
-                                                                "there is no Ballerina.toml file. Run " +
+                    throw LauncherUtils.createLauncherException("no Ballerina.toml file " +
```

Comment:
This is now different from what is in https://github.com/ballerina-platform/ballerina-lang/pull/17596/files#diff-b4885eb61207cdeb93c14da32db64bf2R218. Shall we make the error messages uniform?

Reply:
Changed as follows:

```
"you are trying to build/compile a ballerina project but there is no Ballerina.toml file.",
                            "a Ballerina.toml file is required to build/compile a Ballerina project.",
```